SU-HSDO/suhumsci
# SHS-5653: Some content lists don't have spacing when viewing on mobile device

# READY FOR REVIEW

## Summary
- Adds missing bottom margin to grid items in mobile

## Need Review By (Date)
06/14

## Urgency
medium

## Steps to Test
1. Visit the default News View in [Colorful](https://hs-colorful.suhumsci.loc/default-views/news) and [Traditional](https://hs-traditional.suhumsci.loc/default-views/news) reference sites.
2. Confirm that there's a bottom margin in the items (class: `hb-grid__item`) of the first view (HP 3 Across News) in mobile (see image below)
3. Confirm that the margin is not present above 576px

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
